### DISCUSSION TITLE
Disks health

### DISCUSSION THREAD

**Discussion by Eirikr70** [henrygd/beszel#263] [Ideas]
Hello,
Beszel is really a nice app. I have installed it recently and like using it. It provides me the main useful information about my self-hosted server.
I also use Scrutiny to monitor the health of my hard drives. I think it might be nice to integrate a kind of SMART health state of the drives directly into the Beszel dashboard/alerts instead of having a dedicated app.
That of course is a "nice to have" and certainly not a "must have".
What does the community think about it ?
Is it something heavy to add for the maintainer ?
Thanks

**Comment by henrygd** [comment]
I haven't had time to look into this much.

Scrutiny is MIT licensed and written in Go, so it should be possible to adapt a lot of their code. Or even use it as a package.

If anyone wants to experiment with that, please feel free. It may be a while before I can get around to doing it myself.

**Comment by Gaffeltrucken** [comment]
I'll second this! A SMART dashboard with simplyfied data about, SMART test status, temp, capacity and power-on time for each drive would be great! Feels like it would fit in to Beszels hardwaremonitor focus.
